stations/serializers.py

```diff
@@ -1,16 +1,23 @@
-from rest_framework.fields import SerializerMethodField
+from rest_framework.fields import SerializerMethodField, IntegerField
 from rest_framework.serializers import ModelSerializer
 
+from bikes.models import BikeStatus
 from stations.models import Station
 
 
 class StationSerializer(ModelSerializer):
     activeBikesCount = SerializerMethodField()
+    bikeLimit = IntegerField(write_only=True)
 
     class Meta:
-        fields = ("id", "name", "state", "activeBikesCount")
+        fields = ("id", "name", "state", "activeBikesCount", "bikeLimit")
         model = Station
```

Comment:
Admin z brancha `feat/row-108-bike-limit` wysyła w requeście pole `capacity`. Backend ma natomiast pole `bikeLimit` i requesty nie są przetwarzane. Wystarczy tylko podmienić nazwy w tym pliku.

Reply:
Trzeba to zmienić na frontendzie przecież, w specyfikacji będzie `bikeLimit` więc tutaj musi tak zostać.

Reply:
To przepraszam w takim razie, nie wiedziałem że już było ustalone z innymi zespołami.